examples/conf-quickstart/druid/middleManager/runtime.properties

```diff
@@ -11,10 +11,6 @@ druid.indexer.task.baseTaskDir=var/druid/task
 # HTTP server threads
 druid.server.http.numThreads=9
 
-# Processing threads and buffers
-druid.processing.buffer.sizeBytes=256000000
-druid.processing.numThreads=2
```

Comment:
Currently both `druid.indexer.fork.property.druid.processing.numThreads` and `druid.processing.numThreads` would work (all druid.* properties are passed down automatically). A number of deployments depend on the latter form so it should keep working. fwiw, I also think it's prettier than the former, so I wouldn't want to deprecate it.